### ISSUE TITLE
selector-attribute-quotes: Expected quotes around "undefined"

### ISSUE THREAD

**Issue by jovdb** [stylelint/stylelint#1665]
configuration:

```json
{
  "rules": {
    "selector-attribute-quotes": "always"
  }
}
```

with CSS 
```css
/* If first 2 colors are white and black, the next items should be on a new line */
[data-color="#ffffff"]:first-child + [data-color="#000000"] + [data-color] {
	clear: both;
}
```

I get the error:

```
[data-color="#ffffff"]:first-child + [data-color="#000000"] + [data-color] {
                                                             ^ Expected quotes around "undefined"
```

looking at the [documentation](https://github.com/stylelint/stylelint/tree/master/src/rules/selector-attribute-quotes), I don't think this should be an error.

stylelint: v6.9.0

**Comment by jeddy3** [comment]
@jovdb I can't reproduce this in `7.0.2`. Please update to the latest version and try again.

**Comment by jovdb** [comment]
With v7.0.2 it is indeed solved, thanx
